### PR TITLE
Add a "settle-bot" binary

### PR DESCRIPTION
**Pull request by ckamm**
The bot will call settle on all accounts with unsettled pnl where it can receive either the flat or low-health fee.

### CODE REVIEW

File: settle-bot/Dockerfile.liquidator [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Dockerfile for keeper service in Heroku
```
Comment (microwavedcola1) [inline settle-bot/Dockerfile.liquidator:1]: probably also want to copy paste stuff to push settle bot to gcr, see ci-docker-publish.yml, if it doesnt work out of the box, we could ask pan of course

File: settle-bot/.env.example [hunk 2]
```diff
@@ -0,0 +1,4 @@
+RPC_URL=https://mango.devnet.rpcpool.com
```
Comment (microwavedcola1) [inline settle-bot/.env.example:1]: these need to be updated or?

File: settle-bot/src/settle.rs [hunk 3]
```diff
@@ -0,0 +1,315 @@
...
+                {
+                    // no more interesting pairs that would produce fees
+                    break;
+                }
+
+                // add to batch and maybe send
+                batch_processor
+                    .add_and_maybe_send(account_a, account_b)
+                    .await?;
+                negative_settleable.change_priority(&account_b, settleable_b - settleable);
```
Comment (microwavedcola1) [inline settle-bot/src/settle.rs:211]: hmm do we keep on reusing account_a , settleable_a, without shaving off settleable_a after we settle..similar to how we do for account_b?

Comment (ckamm) [reply settle-bot/src/settle.rs:211]: I'll add a note about that. We loop through the positive-pnl accounts exactly once, so modifying these values doesn't matter.

Comment (microwavedcola1) [reply settle-bot/src/settle.rs:211]: ah, but you could have a mm, who has a large position and be +ve pnl, means the distribution won't be smooth right

Comment (ckamm) [reply settle-bot/src/settle.rs:211]: Yeah, if there's an account with lots of +pnl and all accounts with -pnl are small then that could take multiple passes to get settled right now. I'll add that to the note.

Comment (ckamm) [reply settle-bot/src/settle.rs:211]: actually did a nice solution now

**Review by microwavedcola1 — APPROVED** [review]

**Pull request merged**